packages/ripple-test-tools/package.json

```diff
@@ -20,6 +20,7 @@
     "cypress-axe": "^0.12.0",
     "cypress-cucumber-preprocessor": "^2.0.1",
     "cypress-plugin-snapshots": "^1.4.4",
+    "dayjs": "^1.10.5",
     "glob": "^7.1.6",
     "js-yaml": "^3.14.0",
     "moment": "^2.29.1"
```

Comment:
Aware that this couldn't be removed yet as some SDP projects are using moment from this dependency.